src/relentless/simulate/initialize.py

```diff
@@ -44,6 +45,14 @@ def __init__(self, filename, format=None, dimension=None):
         self.format = format
         self.dimension = dimension
 
+    @property
+    def filename(self):
```

Comment:
Document properties:
```suggestion
    def filename(self):
        """str: Absolute path to file to be read."""
```